widgets/src/Request/ProposeForm.jsx

```diff
@@ -357,7 +357,36 @@ return (
                 start_date: `${new Date(state.startDate).getTime()}`,
                 end_date: `${new Date(state.endDate).getTime()}`,
               },
-            }),
+            });
+            Near.call({
+              contractName: "social.near",
+              methodName: "set",
+              args: {
+                data: {
+                  [context.accountId]: {
+                    index: {
+                      graph: JSON.stringify({
+                        key: "vendor/propose",
+                        value: { proposalId: state.projectId.value },
+                      }),
+                      inbox: JSON.stringify({
+                        key: props.accountId,
+                        value: {
+                          type: "vendor/propose",
+                          proposalId: [
+                            state.projectId.value,
+                            cid,
+                          ],
+                          message: state.message,
+                          vendorId: props.accountId,
+                        },
+                      }),
+                    },
+                  },
+                }
+              }
+            })
+          }
```

Comment:
This needs to be part of the same `Near.call` otherwise it will be ignored, we essentially call `Near.call([...])` where the array elements are these two transactions, so we need to also transform the first call to `add_proposal` to a object instead of separate params

Reply:
Done. Thanks.